src/web/components/SlotBodyEnd.tsx

```diff
@@ -20,10 +25,25 @@ const sendOphanInsertEvent = (): void => {
             name: 'remote_epic_test',
             variant: 'api',
         },
-        action: 'INSERT',
+        action,
     };
 
-    window.guardian.ophan.record({ componentEvent });
+    const key = JSON.stringify(componentEvent);
+    if (!ophanEventRegister[key]) {
+        ophanEventRegister[key] = true;
+        console.log(`Sending ${action} event to Ophan`);
+        window.guardian.ophan.record({ componentEvent });
+    }
+};
+
+const logViewRegister: { [key: string]: boolean } = {};
```

Comment:
I don't have a strong feeling either way so take this or leave it but what about a single unified register object?

Reply:
My thinking here is that the semantics around logging views are probably different than sending data to Ophan, and therefore that the registers should be kept separate. It's a good question though. Let's see what Andre/Nic think. I do imagine these will be moved out of this module shortly fwiw.